.github/workflows/deploy.yml

```
@@ -1,18 +1,16 @@
-#  Taken from https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus
 
-name: github pages
+
+name: Deploy
```

P-e préciser que ce workflow ne s'occupe que de la doc ?
Si on arrive avec un regard extérieur on peut s'attendre à ce que ça gère le déploiement plus largement vu le nom.
```suggestion
name: Deploy doc
```

---

Mon idée était de séparer trois workflows (tests unitaires, tests d'intégration, déploiement), chaque workflow pouvant se décliner pour les différentes parties du code : `back`, `front`, `doc`. En l'occurrence le déploiement du `front` et du `back` est configuré via l'intégration Scalingo mais ça aurait pu ne pas être le cas (si on avait déployé le front sur Object Storage par exemple). Je vais ajouter un commentaire pour préciser comment sont déployés le `front` et le `back` dans ce fichier pour que ce soit plus clair.